scripts/plot_audio.py

```diff
@@ -0,0 +1,61 @@
+import wave
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
**suggestion (non-blocking):** add [PEP 723](https://peps.python.org/pep-0723/) header so it can be run nicely [like this](https://astral.sh/blog/uv-unified-python-packaging#single-file-scripts) instead of requirements.txt

Reply:
can then add a pipx shebang as well